backend/src/test/java/gov/cdc/usds/simplereport/api/model/TestEventExportIntegrationTest.java

```diff
@@ -34,7 +33,6 @@
     })
 @WithSimpleReportStandardUser
 class TestEventExportIntegrationTest extends BaseGraphqlTest {
-  @Autowired protected TestDataFactory _dataFactory;
   @Autowired private TestEventService _testEventService;
```

Comment:
This overwrote the `_dataFactory` provided in `BaseGraphQlTest`, but there was no reason to do so.

https://github.com/CDCgov/prime-simplereport/security/code-scanning/74